modules/svgRenderer/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@msagl/svgRenderer",
+  "version": "1.1.5",
+  "description": "Svg renderer for MSAGL in JavaScript",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "type": "module",
+  "files": [
+    "dist",
+    "dist.min.js",
+    "src"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/microsoft/msagljs"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "tsc --build tsconfig.prod.json && npm run build-bundle && npm run build-worker",
+    "build-bundle": "node ../../esbuild.js bundle.ts dist.min.js",
+    "build-worker": "esbuild ./worker.js --outfile=dist/worker.min.js --bundle --minify"
+  },
+  "dependencies": {
+    "@msagl/parser": "^1.1.4",
+    "msagl-js": "^1.1.4",
+    "panzoom": "^9.4.2",
+    "@msagl/commonRenderer": "^1.1.4"
+  },
+  "gitHead": "07dc5720f142d48f0d8f1dd08a49723ae150d6f7"
```

Comment:
Don't need that

Reply:
That's added automatically by lerna (the publishing tool)

Reply:
I think you meant to remove those lines in the example package, because svgRenderer uses panzoom package, and also uses commonRenderer.